controllers/getTopics.controllers.js

```diff
@@ -0,0 +1,8 @@
+const {selectTopics} = require('../models/selectTopics.models')
+
+exports.getTopics = (req, res) => {
+    selectTopics().then((topics) => {
+        //console.log({topics})
```

Comment:
Dont forget to remove console.logs and comments before pushing work up to github